google/google_maps/__manifest__.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2019, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Google Maps',
+    'version': '12.0.1.0.0',
+    'author': 'Open Source Integrators, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/geospatial',
+    'license': 'AGPL-3',
+    'category': 'Extra Tools',
+    'description': """
+Web Google Map and google places autocomplete address form
+==========================================================
+
+This module brings three features:
+1. Allows user to view all partners addresses on google maps.
+2. Enabled google places autocomplete address form into partner
+form view, it provide autocomplete feature when typing address of partner
+""",
+    'depends': [
+        'base_setup',
+        'base_geolocalize',
+    ],
+    'data': [
+        'data/google_maps_libraries.xml',
+        'views/google_places_template.xml',
+        'views/res_partner.xml',
+        'views/res_config_settings.xml'
+    ],
+    'demo': [],
+    'images': ['static/description/thumbnails.png'],
+    'qweb': ['static/src/xml/widget_places.xml'],
+    'installable': True,
+    'uninstall_hook': 'uninstall_hook',
+    'maintainers': [
+        'gityopie',
```

Comment:
@max3903 Is it fine or I have to remove it from here?

Reply:
@murtuzasaleh It depends on @gityopie.

@gityopie Can we count on you to help maintaining it?

Reply:
@max3903  yes, you can. I'll try my best

Reply:
> @murtuzasaleh The module must be at the root of the repo. Please remove WIP in the title and description if you are done.

@maxime As per the @wolfhall  comment I have added a module in OCA/geospatial/google/google_maps

Do you want to add directly to OCA/geospatial repo?